scrapy/exporters.py

```diff
@@ -86,10 +88,10 @@ def __init__(self, file, **kwargs):
 
     def export_item(self, item):
         itemdict = dict(self._get_serialized_fields(item))
-        self.file.write(self.encoder.encode(itemdict) + '\n')
+        self.file.write(to_bytes(self.encoder.encode(itemdict) + '\n'))
 
 
-class JsonItemExporter(JsonLinesItemExporter):
+class JsonItemExporter(BaseItemExporter):
```

Comment:
hm, ok. I was imagining an exporter class that can do both, and having a `stream` constructor argument of some sort to do lines or not.
no big deal.